### PR TITLE
expressions: Extract builtin information functions to builtin_info.go

### PR DESCRIPTION
**Pull request by shenli**

### CODE REVIEW

File: expression/expressions/builtin_info.go [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+import (
+	"github.com/juju/errors"
+	"github.com/pingcap/tidb/context"
+	"github.com/pingcap/tidb/sessionctx/db"
+	"github.com/pingcap/tidb/sessionctx/variable"
+)
+
+// See: https://dev.mysql.com/doc/refman/5.7/en/information-functions.html
+
```
Comment (qiuyesuifeng) [inline expression/expressions/builtin_info.go]: Remove empty line.

Comment (qiuyesuifeng) [inline expression/expressions/builtin_info.go]: Sorry, it is ok here.

File: expression/expressions/builtin_info.go [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+		return nil, errors.Errorf("Missing ExprEvalArgCtx when evalue builtin")
+	}
+	ctx := c.(context.Context)
+	d := db.GetCurrentSchema(ctx)
+	if d == "" {
+		return nil, nil
+	}
+	return d, nil
+}
+
```
Comment (qiuyesuifeng) [inline expression/expressions/builtin_info.go]: Missed reference here?

Comment (shenli) [inline expression/expressions/builtin_info.go]: All functions in the url at line 27.

Comment (qiuyesuifeng) [inline expression/expressions/builtin_info.go]: Gotcha.

**Comment by qiuyesuifeng** [comment]
LGTM

**Comment by siddontang** [comment]
LGTM

**Pull request merged**